lib/matplotlib/dates.py

```diff
@@ -445,14 +451,20 @@ def julian2num(j):
     Parameters
     ----------
     j : float or sequence of floats
-        Julian date(s)
+        Julian date(s) (days relative to 4713 BC Jan 1, 12:00:00 Julian
+        calendar or 4714 BC Nov 24, 12:00:00, proleptic Gregorian calendar)
```

Comment:
I'm not a fan of parentheses.

```suggestion
        Julian date(s); i.e. days relative to 4713 BC Jan 1, 12:00:00 Julian
        calendar or 4714 BC Nov 24, 12:00:00, proleptic Gregorian calendar.
```

> Parentheses ( ( ) ) can be used to insert explanatory or qualifying material that you want to de-emphasize, or information that doesn't fit naturally into the flow of your text. Because they are so jarring to the reader, parentheses should be avoided whenever possible.

https://wmich.edu/writing/punctuation/parentheses

Reply:
I don't know, `; i.e. stuff...` has got to be as jarring as parentheses, if not more so.

Reply:
We disagree, but it's not worth a lengthy discussion.

Reply:
Fair enough - we can discuss, or codify in a style guide if we have one for docs...